### DISCUSSION TITLE
Is it possible use Subscript and superscript in rofi?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [davatorium/rofi#1970] [Q&A]
How to do ?

similar function in word
![image](https://github.com/davatorium/rofi/assets/19517680/f186a1f1-ac22-470e-b306-b9ffcb80ae82)

**Comment by DaveDavenport** [comment]
Not sure what the actual question is? 
It can display it fine:
![rofi-2024-03-28-0850-00000](https://github.com/davatorium/rofi/assets/406245/a49c1244-ee0b-4c35-87e2-80a4e484ce7f)

How to 'enter' it, depends on how you have your keyboard configured.

**Reply by haithium** [reply]
You can use [Pango Markup](https://docs.gtk.org/Pango/pango_markup.html) `sup` and `sub` tags
